Foundation/include/Poco/InflatingStream.h

```diff
@@ -22,11 +22,8 @@
 #include "Poco/BufferedStreamBuf.h"
 #include <istream>
 #include <ostream>
-#if defined(POCO_UNBUNDLED)
-#include <zlib.h>
-#else
-#include "Poco/zlib.h"
-#endif
+
+struct z_stream_s;
```

Comment:
Don't think this really needs to be here. Just do `struct z_stream_s* _pZstr;` in the class (unless I'm not seeing another usage here).